### ISSUE TITLE
Allow to exclude files / folders from backup.

### ISSUE THREAD

**Issue by Joti-prime** [wisp-gg/frontend#412]
### Describe the feature and why you would like to see it.

When creating a backup allow to choose or exclude / include files / folders. 

### Describe the solution you'd like.

Add the option please.

**Comment by Software-Noob** [comment]
This is already possible on a per-egg basis, please see the [wiki](https://www.notion.so/gamepanel/Daemon-Customisation-1073b5d307bb41709b56a2e11b8437c5?pvs=4#ffb1c8cacb3a4c1f8562705ecd3fb316) for more details.

**Comment by Joti-prime** [comment]
Yes, but must be possible from panel. For example if a client only has 10GB of backup and the server size is 10GB but 5GB only of important information. Then he can exclude the rest and have two backups instead of only one.
